docs/integrations/harness/locations.md

```diff
@@ -0,0 +1,33 @@
+---
+id: locations
+title: Harness Locations
+sidebar_label: Locations
+description: Integrating source code stored in Harness Code into the Backstage catalog
+---
+
+The Harness Code integration supports loading catalog entities from a hosted repository. Entities can be added to
+[static catalog configuration](../../features/software-catalog/configuration.md),
+registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+To use this integration, add configuration to your root `app-config.yaml`:
+
+```yaml
+integrations:
+  harness:
+    - host: app.harness.io
+      token: ${HARNESS_CODE_BEARER_TOKEN}
```

Comment:
This supports our API key as well right? Good to point to https://developer.harness.io/docs/platform/automation/api/add-and-manage-api-keys/

Reply:
fixed to use apiKey @OrkoHunter